dcos_launch/cli.py

```diff
@@ -93,14 +93,18 @@ def do_main(args):
         return 0
 
     if args['describe']:
-        print(json_prettyprint(launcher.describe()))
+        description = launcher.describe()
+        if type(description) is str:
```

Comment:
added this here because terraform.describe returns a string instead of a dict

Reply:
Describe really needs to return a dict IMO The major value of that function is you get out a blog that you can reliably and programmatically pull the master and agent IPs from. If you just pump out a string there is no consistency and dcos-launch is not providing any value

Reply:
@cprovencher remember to update this now that describe will not be returning a string

Reply:
thanks. Done